bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/ScriptExtensionModuleProvider.java

```diff
@@ -78,10 +78,15 @@ private Optional<Value> runtimeModule(String name, String scriptIdentifier, Cont
     private Value toValue(Context ctx, Map<String, Object> map) {
         try {
             return ctx.eval(Source.newBuilder( // convert to Map to JS Object
-                    "js",
-                    "(function (mapOfValues) {\n" + "let rv = {};\n" + "for (var key in mapOfValues) {\n"
-                            + "    rv[key] = mapOfValues.get(key);\n" + "}\n" + "return rv;\n" + "})",
-                    "<generated>").build()).execute(map);
+                    "js", """
+                            (function (mapOfValues) {
```

Comment:
Looks like  some not needed empty lines came into this string

Reply:
single line strings with \n have been extended to a multiline string, looks good to me

Reply:
You’re right, should not have looked at it from my iPhone, it looked wrong, but isn’t